pkg/platform/model/checkpoints.go

```diff
@@ -100,6 +102,35 @@ func GqlReqsToMonoCheckpoint(requirements []*gqlModel.Requirement) []*mono_model
 	return result
 }
 
+func GqlReqVersionConstraintsString(requirement *gqlModel.Requirement) string {
```

Comment:
This is similar to https://github.com/ActiveState/cli/blob/07489ff538ad996075fe0ac08fc256312cd3288c/pkg/platform/model/inventory.go#L478, but I could not reconcile these into a single function because their underlying types are different.

Reply:
I was concerned about adding this to the client due to the maintenance overhead this would create. In the end that was the compromise made, but having it in the client twice feels like it makes that problem worse.

Could you make something to standardize the inputs so the logic for this isn't duplicated?

eg. either refactor the function to take its own types, and have some model function that converts the two different requirement types to the desired one, or use an interface. Either way you'd need two separate ways to standardize the requirement types.